src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -271,11 +272,21 @@ public TopicPartitionChannel(
 
     this.enableSchemaEvolution = this.enableSchematization && hasSchemaEvolutionPermission;
 
+    // telemetry and metrics
+    this.snowflakeTelemetryChannelStatus =
```

Comment:
we create this outside of the telemetryServiceV2 null check because JMX metrics relies on this status